CHANGELOG.md

```diff
@@ -5,7 +5,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
-
+### Added
+- `TavilyWebSearchDriver` to utilize Tavily web search SDK.
```

Comment:
Using the web search SDK is kind of a low level detail. Try rephrasing to emphasize that we're using some feature of Tavily's service.